spec/button_spec.rb

```diff
@@ -123,29 +202,65 @@
           end
         end
 
+        it 'raises an ArgumentError when there is no button with that id' do
+          expect{ ShareProgress::Button.find(999999999)}.to raise_error ArgumentError
```

Comment:
@NealJMD quick reminder to replace `ArgumentError` with `ShareProgress::NotFound`, or similar.